projects/1.19/assets/chat-patches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,60 @@
+{
+    "text.chatpatches.title": "聊天补丁设置",
+
+    "text.chatpatches.dateFormat": "要设置日期和时间格式化样式，点我！",
+    "text.chatpatches.formatCodes": "要设置字符格式化代码和获取相关帮助，点我！",
+
+    "text.chatpatches.time": "启用时间戳显示",
+    "text.chatpatches.timeDate": "时间戳文本",
+    "text.chatpatches.timeFormat": "时间戳修饰文本",
+    "text.chatpatches.timeColor": "时间戳颜色",
+    "text.chatpatches.desc.time": "是否在所有消息前面显示时间戳？",
+    "text.chatpatches.desc.timeDate": "格式化为时间戳的文本。",
+    "text.chatpatches.desc.timeFormat": "使用字符串“&”+用于创建时间戳的格式化代码修饰符和修饰文本的格式。字符“$”是上方格式化时间戳选项的占位符。",
+    "text.chatpatches.desc.timeColor": "表示填充的颜色，否则在生成的时间戳中为空。如果要将其与其他格式修饰符一起使用，请在修饰文本选项中使用字符“&r”。",
+
+    "text.chatpatches.hover": "启用悬停显示",
+    "text.chatpatches.hoverDate": "悬停文本",
+    "text.chatpatches.hoverFormat": "悬停修饰文本",
+    "text.chatpatches.hoverColor": "悬停文本颜色",
+    "text.chatpatches.desc.hover": "将鼠标悬停在聊天中的时间戳文本上时，是否显示更加详细的时间描述文本？",
+    "text.chatpatches.desc.hoverDate": "格式化为较长的日期/时间字符串的文本。",
+    "text.chatpatches.desc.hoverFormat": "使用字符串“&”+用于创建详细悬停说明的格式化代码修饰符和修饰文本的格式。字符“$”是上方格式化时间戳选项的占位符，不可缺失。",
+    "text.chatpatches.desc.hoverColor": "表示填充的颜色，否则在生成的悬停文本中为白色。如果要将其与其他格式修饰符一起使用，请在修饰文本选项中使用字符“&r”。",
+
+    "text.chatpatches.counter": "启用消息计数器显示",
+    "text.chatpatches.counterFormat": "消息计数器文本",
+    "text.chatpatches.counterColor": "消息计数器颜色",
+    "text.chatpatches.desc.counter": "是否在消息后显示消息计数器以指示发送了多个重复项？",
+    "text.chatpatches.desc.counterFormat": "添加到消息末尾用来指示已发送的多个重复项的文本。需要字符“$”用来表示重复次数（不可缺失），也支持使用字符“&”+格式化代码修饰符的格式。",
+    "text.chatpatches.desc.counterColor": "表示填充的颜色，否则在生成的重复计数器中为白色。如果要将其与其他格式化修饰符一起使用，请在修饰文本选项中使用字符“&r”。",
+
+    "text.chatpatches.boundary": "启用分隔线",
+    "text.chatpatches.boundaryFormat": "分隔线文本",
+    "text.chatpatches.boundaryColor": "分隔线颜色",
+    "text.chatpatches.desc.boundary": "是否在离开世界/服务器再加入另一世界/服务器后在聊天栏内显示分隔线？",
+    "text.chatpatches.desc.boundaryFormat": "设置格式化分隔线文本。支持字符“&”+格式化代码修饰符的格式，字符“\\n”表示换行符，字符“$”作为世界名称的占位符。",
+    "text.chatpatches.desc.boundaryColor": "表示填充的颜色，否则在生成的分隔线中为纯白色。如果要将其与其他格式修饰符一起使用，请在修饰文本选项中使用字符“&r”。",
+
+    "text.chatpatches.saveChat": "启用聊天日志",
+    "text.chatpatches.shiftChat": "移动聊天界面",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

原文：**Shift chat**
```suggestion
    "text.chatpatches.shiftChat": "聊天界面偏移",
```